### PR TITLE
Smaller thread replies and system messages

### LINKED ISSUE #906: More subtle join/leave messages
A small UX proposal. Right now, the join/leave messages are taking up a _ton_ of space in the room, making it hard to follow conversation over a longer period of time.

My proposal would be to make join/leave messages into smaller, 'inline' messages instead, like this:

selection_421

This would make it easier to 'scan' over them while following a conversation. On multiple subsequent joins/leaves, they could be grouped together with less margin inbetween, like this:

selection_422

The exact spacing inbetween entries would be up for discussion of course, but as a general proposal. I think it'd really help to focus more on the conversation, rather than on room state.
## <bountysource-plugin>

Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/26967239-more-subtle-join-leave-messages?utm_campaign=plugin&utm_content=tracker%2F16890630&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F16890630&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### LINKED ISSUE #14107: Update the ICON for edited messages and remove avatar
image

The icon should be just a pencil, without the outer box, and now Avatar, the user info is already on the hoover.

Make the icon red, when edited by someone else.

### PR DESCRIPTION
**Pull request by tassoevan**
Closes #14107
Closes #906
Closes #11180

### CODE REVIEW

**Comment by tassoevan** [comment]
Screenshot_20190411-213900
Need some work for mobile (and probably RTL) before be able to merge.

File: app/ui-message/client/message.html [hunk 1]
```diff
@@ -21,64 +21,65 @@
...
+					</span>
+				{{/if}}
+				{{#if sentByEmail}}
+					<span class="sent-by-email">
+						<i class="icon-mail" title='{{_ "Message_sent_by_email"}}' aria-label='{{_ "Message_sent_by_email"}}'></i>
+					</span>
+				{{/if}}
+				{{#if edited}}
+					<span class="edited" title='{{_ "edited"}} {{_ "at"}} {{editTime}} {{_ "by"}} {{editedBy}}'>
+						<i class="icon-pencil{{#if $neq editedBy msg.u.username}} error-color{{/if}}" aria-label="{{_ "Edited"}}"></i>
```
Comment (tassoevan) [inline app/ui-message/client/message.html]: Maybe we can use the SVG icon here.

Comment (engelgabriel) [reply app/ui-message/client/message.html]: We will change all icons to fonts, so leave it.

**Comment by engelgabriel** [comment]
@tassoevan @rodrigok please make sure we use this new template for all system messages

**Comment by rodrigok** [comment]
image

**Pull request merged**